test/webaudio/audioworklet_test_shared.inc

```diff
@@ -83,13 +93,42 @@ void initialised(EMSCRIPTEN_WEBAUDIO_T context, bool success, void* data) {
 }
 
 // Common entry point for the mixer tests
-int main() {
-  static char workletStack[AUDIO_STACK_SIZE];
+// stackSize - audio worklet stack size in bytes
+// return true if the requested memory could be allocated
+bool _main_(int stackSize) {
```

Comment:
Can we just leave the this `int main() {` and use `AUDIO_STACK_SIZE` directly?

Having both `AUDIO_STACK_SIZE` in addition to `stackSize` seems like needless complexity.

Also having both a bool-returning and int-returning version of main seems needless different to other tests that just use a normal `int main`.